selftests/test_testsuite.py

```diff
@@ -503,6 +503,7 @@ def test_skipped_not_meet_req(self) -> None:
             result.check_results.reasons,
         )
         result = self.case_results[1]
+        print(self.default_env.capability)
```

Comment:
remove it.

Reply:
I am still iterating on this PR. But I will mark the PR as reviewable once I push final changes.